contrib/verify-commits/trusted-keys

```diff
@@ -1,3 +1,4 @@
 71A3B16735405025D447E8F274810B012346C9A6
-3F1888C6DCA92A6499C4911FDBA1A67379A1A931
+133EAC179436F14A5CF1B794860FEB804E669320
 32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC
+B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B
```

Comment:
Hmm, these keys are most likely useless for us, as they refer to Bitcoin devs. We should consider using our own keys here in the future.